### DISCUSSION TITLE
Announcing end-of-support for AWS SDK for Java 1.x effective December 31, 2025

### DISCUSSION THREAD

**Discussion by debora-ito** [aws/aws-sdk-java#3078] [Announcements]
**Update (January 6, 2026)**: the AWS SDK for Java 1.x has reached end-of-support.

We [announced](https://aws.amazon.com/blogs/developer/announcing-end-of-support-for-aws-sdk-for-java-v1-x-on-december-31-2025/) the upcoming end-of-support for AWS SDK for Java (v1). We recommend that you migrate to [AWS SDK for Java v2](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/home.html). For dates, additional details, and information on how to migrate, please refer to the linked announcement. If you need assistance or have feedback, please comment in this discussion thread or create an [issue](https://github.com/aws/aws-sdk-java-v2/issues).

**Comment by abhishekpankaj54** [comment]
Will our existing application that uses AWS SDK for Java 1.x stop working after December 31, 2025? if not then how long we can use SDK java V1.x

**Reply by debora-ito** [reply]
The [announcement](https://aws.amazon.com/blogs/developer/announcing-end-of-support-for-aws-sdk-for-java-v1-x-on-december-31-2025/) has an answer for this and more:

Existing applications that use AWS SDK for Java v1.x will continue to function as intended, unless there is a fundamental change to how an AWS service works. 

Since Java SDK v1 will no longer receive updates, we highly recommend to migrate to Java SDK v2.
